browser/ui/webui/brave_rewards/rewards_page_handler.h

```diff
@@ -10,6 +10,7 @@
 #include <string>
 
 #include "base/memory/raw_ptr.h"
+#include "brave/components/brave_ads/core/mojom/brave_ads.mojom-forward.h"
```

Comment:
Not sure if this is necessary since `brave_ads::mojom::AdType` is already brought in by `rewards_page.mojom`.

Reply:
Fixed. Thanks